app/src/main/java/com/pavelkortp/socialnetwork/LoginActivity.kt

```diff
@@ -0,0 +1,58 @@
+package com.pavelkortp.socialnetwork
+
+import android.app.ActivityOptions
+import android.content.Intent
+import android.os.Bundle
+import android.transition.Slide
+import androidx.appcompat.app.AppCompatActivity
+import com.pavelkortp.socialnetwork.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.LoginBTN.setOnClickListener {
+            login()
+        }
+
+        binding.SignUpText.setOnClickListener {
+            signUp()
+        }
+
+        window.enterTransition = Slide()
+        window.exitTransition = Slide()
+    }
+
+    private fun login(): Boolean {
+        val email = binding.EmailInput.text.toString()
+        val pass = binding.PasswordInput.text.toString()
+
+        if (email.isBlank()) {
```

Comment:
this code is repeated in the activity, maybe it makes sense to create a class for validation